api.yml

```
@@ -42,13 +42,13 @@ paths:
           description: invalid input!
         '500':
           description: error!
-  /credential:
+  /credentials/createAndIssue:
```

```suggestion
  /credentials/createAndIssue:
```

I thought we had already agreed to call this .../compositions/ or .../credentials/compositions